### DISCUSSION TITLE
Fixed version of webview2

### DISCUSSION THREAD

**Discussion by aniljbhatia** [tauri-apps/tauri#2754] [General]
Hi.. Tauri looks perfect for my project but some target users will not have access to install webview2 on their machines.
Is it possible to bundle it with the fixed version?

**Comment by FabianLars** [comment]
tracking issue: https://github.com/tauri-apps/tauri/issues/1378
Also, it kinda looks like that the bindings crate ([webview2](https://crates.io/crates/webview2)) doesn't support the fixed version at all, but maybe that's just unlucky wording in the readme 🤔 
But this crate got dropped on `next` branch anyway - replaced by [webview2-com](https://github.com/wravery/webview2-rs). Don't know if this improves the situation tho.
